modular_skyrat/master_files/code/modules/awaymission/mission_code/black_mesa.dm

```diff
@@ -605,6 +605,7 @@
 	prompt_name = "a tactical squad member"
 	outfit = /datum/outfit/hecu
 	you_are_text = "You are an elite tactical squad deployed into the research facility to contain the infestation. DO NOT TRY TO EXPLORE THE LEVEL. STAY AROUND YOUR AREA."
+	restricted_species = /datum/species/human
```

Comment:
mind applying this to the primal plantperson and the ashwalker spawns?

Reply:
sure